packages/cli/src/cli/config/action.bundle.ts

```diff
@@ -41,9 +41,8 @@ export class CommandBundle extends CommandLineAction {
     const config = this.config.value ?? DefaultConfig;
     const bundle = this.output.value ?? DefaultOutput;
     const mem = await ConfigJson.fromPath(config, logger);
+    if (this.assets.value) mem.assets = this.assets.value;
```

Comment:
assets locations were not being hashed as part of the config, this changes that.